usrsctplib/netinet/sctp_input.c

```diff
@@ -1264,7 +1264,8 @@ sctp_handle_error(struct sctp_chunkhdr *ch,
 				asoc->stale_cookie_count++;
 				if (asoc->stale_cookie_count >
 				    asoc->max_init_times) {
-					sctp_abort_notification(stcb, 0, 0, NULL, SCTP_SO_NOT_LOCKED);
+					sctp_abort_notification(stcb, 0, SCTP_CAUSE_STALE_COOKIE, NULL,
+					              SCTP_SO_NOT_LOCKED);
```

Comment:
Hmm. Shouldn't this be handled the same way has too many retransmits?

Reply:
I removed that part from the PR, if you can think of a better error value to provide, it can be added in a later change I guess.